L/libsasl2/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libsasl2"
```

Comment:
Sorry, just noticed: the [name of the package](https://docs.binarybuilder.org/stable/building/#Name) should typically be the name of the upstream project, not that of a "product" (e.g. a library provided).  So I think this should be called something like `CyrusSASL`?  Note that if you change `name` you'll have to rename the directory accordingly.

Reply:
OK thanks, will do